osu.Game/Graphics/ScreenshotManager.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace osu.Game.Graphics
 {
-    public class ScreenshotManager : Container, IKeyBindingHandler<GlobalAction>, IHandleGlobalKeyboardInput
+    public class ScreenshotManager : Drawable, IKeyBindingHandler<GlobalAction>, IHandleGlobalKeyboardInput
```

Comment:
What you really want it to be is a `Component`:
```suggestion
    public class ScreenshotManager : Component, IKeyBindingHandler<GlobalAction>, IHandleGlobalKeyboardInput
```

Reply:
Yes, you are correct. I forgot about this class. :(